packages/text-area/src/vaadin-text-area.js

```diff
@@ -223,6 +226,10 @@ export class TextArea extends ResizeMixin(PatternMixin(InputFieldMixin(ThemableM
       }),
     );
     this.addController(new LabelledInputController(this.inputElement, this._labelController));
+
+    this._tooltipController = new TooltipController(this);
```

Comment:
Should the tooltip move as the text area grows?

https://user-images.githubusercontent.com/1222264/188580136-0cd0a280-aa65-479c-b73b-8500c64b5f95.mp4

Reply:
Good catch. Yes, I think we should add `ResizeObserver` to re-position the tooltip. I will create an issue.

Reply:
If text-area is the only affected component, we could consider working around the issue by adding something like this to `_updateHeight()`
```js
this._tooltipController.setTarget(null);
this._tooltipController.setTarget(this);
```

Reply:
Pretty much any component with a tooltip could be affected e.g. when placed in `vaadin-split-layout`, as in theory there might be some app logic responsible for changing splitter position while focus is still in the field.

Reply:
Also, for tooltips in `manual` mode it's possible to resize `vaadin-split-layout` manually while tooltip is opened.

Reply:
Fixed in #4510 and rebased this PR.